server/matcher/match.go

```diff
@@ -155,11 +155,11 @@ func (m *Matcher) Match(book Booker, queue []*OrderRevealed) (seed []byte, match
 
 	// Track initially unmatched standing limit orders and remove them if they
 	// are matched down-queue so that they aren't added to the nomatched slice.
-	nomatchLimits := make(map[order.OrderID]*OrderRevealed)
+	nomatchStanding := make(map[order.OrderID]*OrderRevealed)
```

Comment:
I decided to go back to the old way here, and use the `else` block below to only use this for standing limits.